### ISSUE TITLE
Cannot run testCafe with HTTPS

### ISSUE THREAD

**Issue by doronsever** [DevExpress/testcafe#2924]
### Are you requesting a feature or reporting a bug?

BUG

### What is the current behavior?
When trying to tun testcafe using HTTPS, the browser always says: 
localhost uses an unsupported protocol.
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

### What is the expected behavior?
Running testcafe on HTTPS

### How would you reproduce the current behavior (if this is a bug)?

I'm trying to set up my local env to use HTTPS.
The reason is that I'm using Angular and it produces source maps and also images are not loaded.
If I'm using testcafe via plain HTTP, I get an error when the tests finish saying: `Error [ERR_INVALID_PROTOCOL]: Protocol "http:" not supported. Expected "https:"`

I went over this tutorial: http://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html and trying to use it via the cli but without success.

I tried creating a local PFX file, or using public key + cert. Non will work.

For example, I went over this tutorial: https://medium.com/@jonsamp/how-to-set-up-https-on-localhost-for-macos-b597bcf935ee

And then run this testcafe command:
`testcafe  chrome e2e/tests/analyze/ --ssl key=~/.localhost-ssl/localhost.key cert=~/.localhost-ssl/localhost.crt --allow-insecure-localhost --allow-running-insecure-content --dev --hostname=localhost
`

But still the same result.

Is it a bug or am I doing something wrong?

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system: macOS
* testcafe version: 0.22
* node.js version: 9.7.1

**Comment by doronsever** [comment]
@miherlosev this doesn't help also...
I also tried using the crt+key from browser-sync just like I'm using when running my Angular localhost on https but without success...

`testcafe  'chrome --allow-insecure-localhost' e2e/tests/analyze/ --hostname=localhost --ssl key=/node_modules/browser-sync/lib/server/certs/server.key;cert=/node_modules/browser-sync/lib/server/certs/server.crt`

**Comment by miherlosev** [comment]
Could you please provide a simple example that I can run on my computer?

**Comment by doronsever** [comment]
Sure, which example do you need?
I paste you how I run the command in the previous reponse

**Comment by miherlosev** [comment]
Application sources, test code and certificates.

**Comment by doronsever** [comment]
@miherlosev I've created an empty Angular project by your request.
https://github.com/doronsever/my-tcf-ssl
Just clone it, run `npm install`
And then run your command `testcafe "chrome --allow-insecure-localhost --allow-running-insecure-content" e2e/tests/analyze/ --ssl key=./certs/localhost.key;cert=./certs/localhost.crt  --dev --hostname=localhost
`

**Comment by miherlosev** [comment]
Ok. I am investigating the problem

**Comment by miherlosev** [comment]
Yes, it's a bug. We will fix it in the [current development sprint](https://github.com/DevExpress/testcafe/milestone/34).

**Comment by doronsever** [comment]
Great, it will also be great if you can improve the documentation on how to use it.
I would never have guessed that the right way to use the SSL params will be `--ssl key=./certs/localhost.key;cert=./certs/localhost.crt `

**Comment by miherlosev** [comment]
The fix is already in [PR](https://github.com/DevExpress/testcafe/pull/2927).

> I would never have guessed that the right way to use the SSL params will be --ssl key=./certs/localhost.key;cert=./certs/localhost.crt

We have an [example](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--ssl-options) of using the `--ssl` option  and a [help topic](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html) that describes the main concepts.

**Comment by doronsever** [comment]
> We have an [example](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--ssl-options) of using the `--ssl` option and a [help topic](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html) that describes the main concepts.

I think it will be more clear how to implement it with a dedicated example. Friendly advice :)

**Comment by doronsever** [comment]
@miherlosev I still cannot use local variables after upgrading to 0.23.0

I'm running: ` testcafe 'chrome --allow-insecure-localhost' e2e/tests/analyze/ --hostname=localhost --ssl key=/node_modules/browser-sync/lib/server/certs/server.key;cert=/node_modules/browser-sync/lib/server/certs/server.crt`

And getting:
> Error: error:0906D06C:PEM routines:PEM_read_bio:no start line

**Comment by LavrovArtem** [comment]
Related to https://github.com/DevExpress/testcafe-hammerhead/issues/1806

**Comment by kaitmore** [comment]
@doronsever 
I was experiencing this same issue using a pfx cert. I realized that testcafe was only picking up the first option passed to the --ssl flag, (in your case, 'key'). If I wrapped the whole thing in quotes like, `--ssl 'key=path/to/key;cert=path/to/cert;'` it works!

**Comment by doronsever** [comment]
Wow...
OK, I will check it out. Thanks!
@miherlosev another reason why you should add this to your docs :)

**Comment by miteshsavani** [comment]
@kaitmore It is not working for me even if i put the quotes like `--ssl 'key=path/to/key;cert=path/to/cert;'` i am facing same issue 
`This site can’t provide a secure connection localhost uses an unsupported protocol.
ERR_SSL_VERSION_OR_CIPHER_MISMATCH`
